src/bindings/html/langs.js

```diff
@@ -18,7 +18,7 @@ export function getAdditionalLanguages() {
 export function onlanguagechage(
   appVersion, defaultLang, availableLangs, requestedLangs) {
 
-  return this.languages = Promise.all([
+  return Promise.all([
```

Comment:
There's a difference between assigning to `this.languages` here and in `changeLanguage`. Here `this.languages` is the new promise immediately after `onlanguagechange` is called.  In `changeLanguage` it's the old promise until `Promise.all()` is resolved asynchronously.  I think that would be a bug because `document.l10n.format*` would return translations in a non-deterministic language, depending on how long it takes for `Promise.all()` to resolve.